storageFormats/eventMetadata.h

```diff
@@ -11,6 +11,8 @@ class TTree;
 
 
 namespace rpwa {
+	typedef std::map<std::string, std::pair<double, double> > binningMapType;
+	typedef std::map<std::string, double > multibinCenterType;
```

Comment:
Maybe "binCenterType" would be a better name for "multibinCenterType", beause it just holds variable name and bin center. Not sure we should go that far, but in principle "binRangeType" would be a better name for "binningMapType". However this would probably require changes in a lot of places.